lib/Transforms/SecretInsertMgmt/SecretInsertMgmtBGV.cpp

```diff
@@ -87,6 +92,37 @@ struct SecretInsertMgmtBGV
     pipeline.addPass(createCSEPass());
     pipeline.addPass(mgmt::createAnnotateMgmt());
     (void)runPipeline(pipeline, getOperation());
+
+    // The following part is experimental noise analysis
+    // for BGV. Should observe the result using --debug-only=NoiseAnalysis
+    // and --debug-only=secret-insert-mgmt-bgv
```

Comment:
```suggestion
    // for BGV. Should observe the result using --debug-only=NoiseAnalysis,secret-insert-mgmt-bgv
```

FYI these flags accept a comma-separated list of debug-type ids.